app/views/index.html

```diff
@@ -168,6 +168,38 @@ <h3 class="govuk-heading-m">
         </a>
       </ul>
 
+      <h2 class="govuk-heading-l">
+        Sprint 7
```

Comment:
I think `sprint-7` naming is fine in code, but perhaps will confuse others viewing this page later - perhaps “Sprint 6 - final iterations”?